c10/util/C++17.h

```diff
@@ -72,7 +72,7 @@ constexpr inline T&& forward(guts::remove_reference_t<T>&& t) noexcept {
 template <typename T, typename... Args>
 typename std::enable_if<!std::is_array<T>::value, std::unique_ptr<T>>::type
 make_unique(Args&&... args) {
-  return std::unique_ptr<T>(new T(forward<Args>(args)...));
+  return std::unique_ptr<T>(new T(c10::guts::forward<Args>(args)...));
```

Comment:
these changes will be removed before merging, fixed as part of https://github.com/pytorch/pytorch/pull/18492